package.json

```diff
@@ -0,0 +1,82 @@
+{
+  "name": "benefiaciary-registration-portal",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "start": "set PORT=3007 && react-app-rewired start",
```

Comment:
Is it necessary to hardcode port like this?  What if a developer needs to run UI on another port?

Reply:
I will address this.